support-workers/src/main/scala/com/gu/support/workers/lambdas/Handler.scala

```diff
@@ -36,12 +59,17 @@ abstract class Handler[IN, OUT](implicit
     }
 
   def handleRequestFuture(is: InputStream, os: OutputStream, context: Context): Future[Unit] = {
+    def log[T](message: String, a: T): Unit = {
```

Comment:
Maybe for a future PR - but would we want to pull this into our `SafeLogger` so we benefit from this everywhere?

Reply:
good point.  Yes it would make so much sense to do that, I'll do that in another PR.